src/components/dashboard/main-content/course-enrollments/CourseEnrollments.jsx

```diff
@@ -110,6 +110,13 @@ const CourseEnrollments = ({ children }) => {
     );
   }
 
+   const handleOnClose = () => {
+    // 1) setShowCancelledAssignmentsAlert(false);
+    // 2) patchLearnerContentAssignment
+    // 3) refetch fetchRedeemableLearnerCreditPolicies to update list
```

Comment:
[clarification] The policies/assignments data is fetched via the `useRedeemableLearnerCreditPolicies` hook (`useQuery`). When you say refetch `fetchRedeemableLearnerCreditPolicies`, we could likely invalidate the policies/assignments query which would force it to refetch in the background?

Also note, that @brobro10000 is refactoring the `useRedeemableLearnerCreditPolicies` a bit to make use of a Query Key Factory pattern ([docs](https://github.com/openedx/frontend-app-admin-portal/blob/master/docs/decisions/0006-tanstack-react-query.rst)) as used in his [PR](https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/899#discussion_r1425666144).